app/managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema_db.db3")
+        self.table_name = "actors"
+
+    def all(self) -> list[Actor]:
+        actors_coursor = self._connection.execute(
+            "SELECT * FROM actors"
```

Comment:
you have `{self.table_name}` everywhere but you used `actors` here

Reply:
after adding self.table_name I forgot to change this.
Thank's, fixed